snippets/fields-pubnote.md

```
@@ -0,0 +1,19 @@
+<!-- stub — regenerated once OpenAPI spec ships SearchPubnoteResponse -->
+
+| Field                           | Type   |
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)